kong/plugins/prometheus/exporter.lua

```diff
@@ -298,19 +316,9 @@ local function metric_data()
   end
 
   if ngx.location then
-    local r = ngx.location.capture "/nginx_status"
-
-    if r.status ~= 200 then
-      kong.log.warn("prometheus: failed to retrieve /nginx_status ",
-        "while processing /metrics endpoint")
-
-    else
-      local accepted, handled, total = select(3, find(r.body,
-        "accepts handled requests\n (%d*) (%d*) (%d*)"))
-      metrics.connections:set(accepted, { "accepted" })
-      metrics.connections:set(handled, { "handled" })
-      metrics.connections:set(total, { "total" })
-    end
+    metrics.connections:set(tonumber(C.ngx_stat_accepted[0]), { "accepted" })
```

Comment:
`tonumber` should be localized in module scope.

Reply:
moved to PDK/nginx.lua now.